### ISSUE TITLE
Impossible to setup Email only signup with cli

### ISSUE THREAD

**Issue by giulioambrogi** [aws-amplify/amplify-cli#8061]
### Before opening, please confirm:

- [X] I have installed the latest version of the Amplify CLI (see above), and confirmed that the issue still persists.
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-cli/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### How did you install the Amplify CLI?

npm

### If applicable, what version of Node.js are you using?

12.16.1

### Amplify CLI Version

5.3.0

### What operating system are you using?

mac

### Amplify Categories

auth

### Amplify Commands

add

### Describe the bug

# Sign up by email

I want my app to allow sign ups by email (no username required).

So I followed [your docs](https://docs.amplify.aws/guides/authentication/email-only-authentication/q/platform/js/
) step by step but I get this error when trying to sign up: _Username cannot be of email format, since user pool is configured for email alias_. See details below.

## This is how my user pool looks like 

I would expect to see the second radio option ticket instead
![Screenshot 2021-08-30 at 14 05 57](https://user-images.githubusercontent.com/11599158/131343644-491c2fc1-dd87-46e8-81a9-b909770a40a1.png)

![Screenshot 2021-08-30 at 13 50 50](https://user-images.githubusercontent.com/11599158/131342615-f6bceafc-090d-4180-b1c4-682f101df7c4.png)

### Expected behavior

I'm able to sign up just by providing email and password with no error.

### Reproduction steps

1. From  [your docs](https://docs.amplify.aws/guides/authentication/email-only-authentication/q/platform/js/
)
2. 
```
amplify add auth

? Do you want to use the default authentication and security configuration? Default configuration
? How do you want users to be able to sign in? Email
? Do you want to configure advanced settings? No, I am done.
```
3. amplify push
4. 
```
import { AmplifyAuthenticator } from '@aws-amplify/ui-react';

<AmplifyAuthenticator usernameAlias="email" />
````

### Log output

{code: "InvalidParameterException", name: "InvalidParameterException", message: "Username cannot be of email format, since user pool is configured for email alias."}

### Additional information

I bootstrapped my project using your docs, you can try on a fresh react / next project.

**Comment by giulioambrogi** [comment]
Looks like it's related to this issue: https://github.com/aws-amplify/amplify-cli/issues/7833#issuecomment-889596331

Temporary fix: downgrade to 5.1.2.

I'll let the admins close this one

**Comment by cjihrig** [comment]
Sorry for the inconvenience. I'll close this as being tracked in #7833.
